app/views/admin/navigation_links/index.html.erb

```diff
@@ -2,26 +2,59 @@
   <div class="my-5"> Configure the left sidebar navigation links that appear at <a href="<%= app_url %>"><%= app_url %></a> using the form below:</div>
 
   <%= render "add_navigation_link_modal" %>
-  <% @navigation_links.each do |link| %>
-    <div class="flex pt-5 items-center">
-      <div class="mr-5"><%= link[:position] %></div>
-      <div class="admin-navigation-links">
-        <a href="<%= link[:url] %>" class="crayons-link crayons-link--block bg-base-20">
-          <div class="crayons-icon crayons-icon--default">
-              <%= link[:icon].html_safe %>
+  <div class="pt-8">
+    <div class="fw-bold">
+      <%= community_name %>
+    </div>
```

Comment:
accessibility (blocking) - I think we should make this and the "Other" heading `h2` headings - this will help assistive tech users get a sense of the document outline of the page, and be able to skip right to the section they're looking for 🙂